code/parachain/frame/pablo/src/lib.rs

```diff
@@ -985,21 +986,27 @@ pub mod pallet {
 
 		fn spot_price(
 			pool_id: Self::PoolId,
-			base_asset: AssetAmount<Self::AssetId, Self::Balance>,
```

Comment:
Bit of a change of mind here, I think we can go with the existing naming for this function as this a specific place where this definition make sense. As someone is asking for quote on the base asset.